project/MimaExcludes.scala

```diff
@@ -82,6 +82,10 @@ object MimaExcludes {
             // SPARK-5166 Spark SQL API stabilization
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Transformer.transform"),
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Estimator.fit")
+          ) ++ Seq(
+            // SPARK-5315 Spark Streaming Java API returns Scala DStream
+            ProblemFilters.exclude[MissingMethodProblem](
+              "org.apache.spark.streaming.api.java.JavaDStreamLike.reduceByWindow")
```

Comment:
This actually not a false positive. Since JavaDStreamLike is a trait, it boils down to an abstract class. If someone has inherited their own class from DStreamLike, and had a function reduceByKeyAndWindow with this signature (the new one), his code will break as his code will now have to "override" the method. So technically it is correctly. But I dont envision any one doing this, so this binary compatibility break is fine.